go/vt/vttablet/tabletmanager/restore.go

```diff
@@ -191,6 +192,7 @@ func (tm *TabletManager) restoreDataLocked(ctx context.Context, logger logutil.L
 		Shard:               tablet.Shard,
 		StartTime:           logutil.ProtoToTime(request.BackupTime),
 		DryRun:              request.DryRun,
+		Stats:               backupstats.RestoreStats(),
```

Comment:
If `Stats` is not set then `mysqlctl` will use `backupstats.NopStats`. This way any out-of-tree code can opt in to the new stats, or not (and not have to worry about nil deference errors).